doc/manual/io.rst

```diff
@@ -263,16 +263,14 @@ European data format (.edf)
 
 EDF and EDF+ files can be read in using :func:`mne.io.read_raw_edf`.
 
-http://www.edfplus.info/specs/edf.html
+EDF (European Data Format; seehttp://www.edfplus.info/specs/edf.html) and EDF+
+(http://www.edfplus.info/specs/edfplus.html) are 16-bit formats
```

Comment:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets

I would use the "embedded URI" version here